canonical-websites/www.ubuntu.com
# Remove server/plans-and-pricing page for support version

## Done

* Removed duplicative /server/plans-and-pricing page
* Updated link on hyperscale and updated the [copy doc](https://docs.google.com/document/d/1f56CCzH519lIVa4KvxDt0QpxYFMK-bMdVlKo4TvPSIk/edit)
* Removed the page from the navigation
* Added a redirect to /support/plans-and-pricing

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser
- see that Plans and pricing has been removed from the navigation
- see that [hyperscale](http://0.0.0.0:8001/server/hyperscale) links to  /support/plans-and-pricing
- see that [server/plans-and-pricing](http://0.0.0.0:8001/server/plans-and-pricing) goes to /support/plans-and-pricing
